### PR TITLE
testing with PIT

### PR DESCRIPTION
**Pull request by diamandv**
this test causes an error:
PIT >> WARNING : Minion exited abnormally due to MEMORY_ERROR

//Task 18
@test
void givenIntegersWithMaximumArraySize_whenGetRandomArrayWithBounds_thenReturnIntegersFromLowerToUpperBoundsWithCertainLength() throws Exception {
//given
Runtime r = Runtime.getRuntime();
int maxSizeOfArray = (int) ((r.maxMemory() / 4) * 0.95);
int lowerBorder = 2;
int upperBorder = 25;
//when
int[] actualIntegers = TwentyMethods.getRandomArrayWithBounds(maxSizeOfArray, lowerBorder, upperBorder);
//then
assertEquals(maxSizeOfArray, actualIntegers.length);
}

### CODE REVIEW

File: src/main/java/org/twentymethods/TwentyMethods.java [hunk 1]
```diff
@@ -152,7 +152,7 @@ public static int findIndexOfIntegersFromTheEndThatMatchesWithItsValue(int[] int
         }
         for (int i = integers.length - 1; i >= 0; i--) {
             if (integers[i] == intToFind) {
-                return i;
+                return integers.length - i - 1;
             }
         }
         return a;
```
Comment (BogdanParashchak) [inline src/main/java/org/twentymethods/TwentyMethods.java:158]: to be in separate issue/branch/PR

File: src/test/java/org/twentymethods/TwentyMethodsTests.java [hunk 2]
```diff
@@ -487,7 +487,7 @@ void givenMatchIntIndexAndIntArray_whenFindIndexOfIntegersFromTheEndThatMatchesW
         //when
         int actualIndex = TwentyMethods.findIndexOfIntegersFromTheEndThatMatchesWithItsValue(integers, intToFind);
         //then
-        assertEquals(4, actualIndex);
+        assertEquals(0, actualIndex);
         assertNotEquals(2, actualIndex);
     }
```
Comment (BogdanParashchak) [inline src/test/java/org/twentymethods/TwentyMethodsTests.java:493]: to be in separate issue/branch/PR

File: src/test/java/org/twentymethods/TwentyMethodsTests.java [hunk 3]
```diff
@@ -1379,4 +1379,4 @@ void givenTwoArraysOfCharsWithSecondEmpty_whenCheckIfIsSecondInFirstWithUsingStr
         //then
         assertTrue(isSequence);
     }
-}
+}
```
Comment (BogdanParashchak) [inline src/test/java/org/twentymethods/TwentyMethodsTests.java:1382]: ????

**Review by BogdanParashchak — CHANGES_REQUESTED** [review]

File: src/test/java/org/twentymethods/TwentyMethodsTests.java (hunk 3 above)
Comment (diamandv) [reply src/test/java/org/twentymethods/TwentyMethodsTests.java:1382]: resolved

**Review by BogdanParashchak — APPROVED** [review]

**Pull request merged**